homeassistant/components/sleepiq/const.py

```diff
@@ -13,6 +13,8 @@
 SLEEP_NUMBER = "sleep_number"
 FOOT_WARMING_TIMER = "foot_warming_timer"
 FOOT_WARMER = "foot_warmer"
+CORE_CLIMATE_TIMER = "core_climate_timer"
+CORE_CLIMATE = "core_climate"
 ENTITY_TYPES = {
     ACTUATOR: "Position",
```

Comment:
These should all be lower case since we don't want Title Case.

Reply:
I am not the original contributor for the Entity_Types. This would not up to me to change, @kbickar would be the one to look at that. 

I also don't believe this should be something that should block this PR. Do you disagree?

Reply:
The strings are converted to lowercase for the entity ID. These are used for both name and ID in that way